qds_sdk/cluster.py

```diff
@@ -216,6 +216,13 @@ def _parse_create_update(cls, args, action):
                              default=None,
                              help="Use hadoop1 instead of hadoop2. This is the default.")
 
+        spark = hadoop_group.add_mutually_exclusive_group()
+        spark.add_argument("--use-spark",
+                           dest="use_spark",
+                           action="store_true",
+                           default=None,
+                           help="Turn on spark for this cluster")
```

Comment:
Valid point.

Though saying `--dont-use-spark` would show their intention pretty clearly. I mean we can do that from the UI by changing the cluster type - so wondering whether the SDK should be crippled in this regard.

I don't have any strong opinions on this other than parity between UI and SDK. So, your decision.